app/src/test/java/org/mozilla/tv/firefox/RxTest.kt

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox
+
+import io.reactivex.subjects.PublishSubject
+import org.junit.Test
+
+/**
+ * This class is for testing combinations of operators to ensure that they work
+ * as we expect.
+ *
+ * Rx is hard.  Don't wing it; test.
+ */
+class RxTest {
```

Comment:
Can we add a lifecycle test here as well? That is important considering we are looking to replace `LiveData` eventually.

Reply:
I'm disposing manually in the `Fragment`, so this should be identical to how we're using it in practice.  I suppose we could add a test to the `Fragment`, and assert that dispose is called?